### PR TITLE
duration prettier

### PR DESCRIPTION
**Pull request by Quetz4l**
* example: from 128 to 6 * SECONDS + 8 TICKS

### CODE REVIEW

**Comment by boubou19** [comment]
not sure if we want to change them like that, as you are skipping 8 ticks.

File: src/main/java/com/dreammaster/scripts/ScriptAdvancedBackpacks.java [hunk 1]
```diff
@@ -384,7 +385,7 @@ public void loadRecipes() {
 
         GT_Values.RA.stdBuilder().itemInputs(getModItem(Minecraft.ID, "melon", 1, 0, missing))
                 .fluidInputs(FluidRegistry.getFluidStack("water", 750))
-                .fluidOutputs(FluidRegistry.getFluidStack("melonjuice", 750)).duration(128).eut(4)
+                .fluidOutputs(FluidRegistry.getFluidStack("melonjuice", 750)).duration(6 * SECONDS).eut(4)
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/scripts/ScriptAdvancedBackpacks.java:388]: This conversation ist not correct

**Comment by dahgul333** [comment]
some parts are rounded so that the time does not consist of more than 2 parts @boubou19 , @Dream-Master

**Comment by Dream-Master** [comment]
Then round them up not down

**Comment by Quetz4l** [comment]
ah yes, if the duration is more than 100 ticks, the ticks are not written. This is meant to discard useless ticks

**Comment by GDCloudstrike** [comment]
There are a lot of instances where you used x * SECONDS + 10 * TICKS, so why not just keep it as 6 * SECONDS + 8 * TICKS?

**Comment by Quetz4l** [comment]
In the formula with minutes and hours ticks make no sense, they just lengthen the code. you can decide whether it is better to round up or leave it as in the original. tomorrow I will finish the pr

**Comment by dahgul333** [comment]
> There are a lot of instances where you used x * SECONDS + 10 * TICKS, so why not just keep it as 6 * SECONDS + 8 * TICKS? Also, what do you mean by useless ticks? why are they useless?

not all recipes were created with precise parameters, a lot was taken from the head, 8 ticks out of 128 is only 6.2% ,6,2% from a number taken at random, but removes an entire segment

**Comment by GDCloudstrike** [comment]
> 
> not all recipes were created with precise parameters, a lot was taken from the head, 8 ticks out of 128 is only 6.2% ,6,2% from a number taken at random, but removes an entire segment

6.2% is still a significant amount, especially if that is applied to loads of recipes. Scanning through the changes here, it seems that these types of recipes are also comparatively only a small subset of the whole change, so adding those lost ticks back would not significantly impact the code length while keeping accuracy.

**Comment by dahgul333** [comment]
> Then round them up not down

edited 👀

**Comment by Dream-Master** [comment]
did the tick Phrase work? @boubou19 min , sec works.

**Review by boubou19 — APPROVED** [review]
i looked 40 recipes randomly with various durations and they all seems fine.

**Pull request merged**